internal/api/scim/resources/user.go

```diff
@@ -171,6 +178,10 @@ func (h *UsersHandler) Update(ctx context.Context, id string, operations patch.O
 		return err
 	}
 
+	if !userWM.UserState.Exists() {
```

Comment:
Is this not already checked in the `UserHumanWriteModel`?

Reply:
Nope AFAICT it isn't and I didn’t move it there because I wasn’t sure about the potential impact. Is it safe to move it there?

Reply:
You are correct, my bad 🙈 I wasn't careful enough while rebasing...